redhat-developer/developers.redhat.com
# [DEVELOPER-4375] Add indent/outdent to WYSIWYG

This adds indent/outdent functionality to the CKEditor WYSIWYG Red Hat
text format. This includes adding the actual buttons in the text format
of the editor and installing and configuring the ckeditor_indentblock
Drupal contrib module.

### JIRA Issue Link
* https://issues.jboss.org/browse/DEVELOPER-4375

### Verification Process

* Go to any Article (for example) node edit form

* In the Body field, create something that looks like this using the new indent button to the far right of the WYSIWYG to indent some text blocks

<img width="1004" alt="Screen Shot 2019-03-11 at 3 34 49 PM" src="https://user-images.githubusercontent.com/7155034/54162646-73d64380-4413-11e9-9a9f-2d22b84ab937.png">

* Submit the node edit form to persist your changes and view the node you just edited. It should look something like this

<img width="961" alt="Screen Shot 2019-03-11 at 3 34 57 PM" src="https://user-images.githubusercontent.com/7155034/54162647-73d64380-4413-11e9-9665-4c66c9b584a7.png">
